html/modules/custom/reliefweb_form/js/widget.selectionlimit.js

```diff
@@ -71,7 +71,7 @@
       }
 
       // Enable selection limit on relevant checkboxes fields.
-      var elements = context.querySelectorAll('fieldset[data-with-selection-limit]:not([data-with-selection-limit-processed])');
+      var elements = context.querySelectorAll('[data-with-selection-limit]:not([data-with-selection-limit-processed])');
```

Comment:
This means it will bind to all individual input elements as well, I just tried with the previous code and limit is working in page and popup :confused:

Reply:
Ah, I didn't notice that the `data-with-selection-limit` was on the individual input as well as the checkboxes container. That was not the case in RW7 and that's also what the `checkSelectionLimit` function implies with `var inputs = element.getElementsByTagName('input');`.

Created https://humanitarian.atlassian.net/browse/RW-611 to solve the problem at the root and only have the attribute on the checkboxes container and not the individual inputs.